packages/runtime/src/helpers/analysis.ts

```diff
@@ -100,7 +101,7 @@ export const extractConfigFromFile = async (apiFilePath: string, appDir: string)
 
   if (!extractConstValueModulePath || !parseModulePath) {
     if (!hasWarnedAboutNextVersion) {
-      console.log("This version of Next.js doesn't support advanced API routes. Skipping...")
+      logger.log("This version of Next.js doesn't support advanced API routes. Skipping...")
```

Comment:
```suggestion
      logger.info("This version of Next.js doesn't support advanced API routes. Skipping...")
```

`.info` was used as replacement in other places that replace `console.log`. `logger.log` currently cause TS errors:

```
src/helpers/analysis.ts(104,18): error TS2345: Argument of type 'string' is not assignable to parameter of type 'LogEntry'.
```
as `logger.log` has this signature:

```
  interface LogEntry {
    level: string;
    message: string;
    [optionName: string]: any;
  }

  interface LogMethod {
    (level: string, message: string, callback: LogCallback): Logger;
    (level: string, message: string, meta: any, callback: LogCallback): Logger;
    (level: string, message: string, ...meta: any[]): Logger;
    (entry: LogEntry): Logger;
    (level: string, message: any): Logger;
  }
```